packages/legend-query-builder/src/stores/milestoning/QueryBuilderMilestoningHelper.ts

```diff
@@ -316,10 +317,54 @@ export const validateMilestoningPropertyExpressionChain = (
           `Property of milestoning sterotype '${MILESTONING_STEREOTYPE.BITEMPORAL}' should not have more than two parameters`,
         );
       }
+      queryBuilderState.milestoningState.setProcessingDate(
+        propertyExpression.parametersValues[1],
```

Comment:
Why did we need to add this logic to validation stage? Don't we want to throw errors in UI if people do something unexpected in text mode?

Reply:
The changes here are more about initializing `milestongState`. 
If the root class is not milestoning, I found that the `milestongState`  is not initialized at all.